docs/what-is-rocm.md

```diff
@@ -37,55 +37,58 @@ ROCm consists of the following drivers, development tools, and APIs.
 | Project | Description |
 | :---------------- | :------------ |
 | [AMD Compute Language Runtimes (CLR)](https://github.com/ROCm-Developer-Tools/clr) | Contains source code for AMD's compute languages runtimes: {doc}`HIP <hip:index>` and OpenCL |
+| {doc}`AMD SMI <amdsmi:index>` | A C library for Linux that provides a user space interface for applications to monitor and control AMD devices |
 | [AOMP](https://github.com/ROCm-Developer-Tools/aomp/) | A scripted build of [LLVM](https://github.com/RadeonOpenCompute/llvm-project) and supporting software |
 | [Asynchronous Task and Memory Interface (ATMI)](https://github.com/RadeonOpenCompute/atmi/) | A runtime framework for efficient task management in heterogeneous CPU-GPU systems |
```

Comment:
```suggestion
```
Maybe we can remove this, it doesn't seem to be included in ROCm anymore as of 5.5.1